SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Please report vulnerabilities to this repository via GitHub security advisories instead.
+
+How? Inside affected repository --> security tab --> advisories --> _New draft security advisory_
+
+In severe cases, you can also report a found vulnerability here:
```

Comment:
That is some quite weird wording. I would just list the means of communication to report a vulnerability in order of your own preference.

A template SECURITY.md file can be found here: https://github.com/eclipse-csi/security-handbook/blob/main/templates/SECURITY.md